app/services/flood_risk_engine/find_address_by_uprn.rb

```diff
@@ -0,0 +1,31 @@
+module FloodRiskEngine
+  class FindAddressByUprn
+
+    attr_reader :address_data
+
+    def initialize(uprn)
+      Rails.logger.debug("FindAddressByuPRN supplied with params [#{uprn.inspect}]")
+      @uprn = uprn
+      @address_data = ""
```

Comment:
I think there needs to be a consistent way of clearing the @address_data. Either `@address_data = ""` or `@address_data = nil`